server/services/caldav/index.js

```diff
@@ -0,0 +1,68 @@
+const Promise = require('bluebird');
+const logger = require('../../utils/logger');
+const CalDAVHandler = require('./lib');
+const CalDAVController = require('./api/caldav.controller');
+
+module.exports = function CalDAVService(gladys, serviceId) {
+  const ical = require('ical');
+  const dav = require('dav');
+  const moment = require('moment');
+  const xmlDom = require('xmldom');
+  const request = require('request-promise-native');
+
+  const calDavHandler = new CalDAVHandler(gladys, serviceId, ical, dav, moment, xmlDom, request);
+
+  let interval;
+
+  /**
+   * @public
+   * @description This function starts the CalDAV service
+   * and start interval to sync all users every 30mn
+   * @example
+   * gladys.services.caldav.start();
+   */
+  async function start() {
+    logger.log('starting CalDAV service');
+    interval = setInterval(async () => {
+      try {
+        const users = await gladys.user.get();
+        const service = await gladys.service.getLocalServiceByName('caldav');
+
+        await Promise.map(
```

Comment:
Don't do all that in the start function ! 

The start function should be clean and concise, try to put that in a separate function so the code can be more easily tested, and more understandable !